lib/server-render.js

```diff
@@ -28,7 +28,7 @@ function serverRender (entry, cb) {
     app = proxyquire(entry, {})
   } catch (err) {
     var failedRequire = err.message === `Cannot find module '${entry}'`
-    if (!failedRequire) return cb(err)
+    if (!failedRequire) render.ssrError = err
```

Comment:
I was not sure if the `failedRequire ` errors should still be filtered in this way, so I left it.

Reply:
Yeah, I think we want to filter them still.